examples/docker-compose.yml

```diff
@@ -1,14 +1,24 @@
 version: '2'
 services:
   conjur:
-    image: registry.tld/conjur-appliance-cuke-master:4.9-stable
+    image: cyberark/conjur:0.1.0-stable
     environment:
-      CONJUR_AUTHN_LOGIN: admin
-      CONJUR_AUTHN_API_KEY: secret
-      CONJUR_ACCOUNT: cucumber
+      CONJUR_ADMIN_PASSWORD: secret
+      DATABASE_URL: postgres://postgres@db
+      CONJUR_DATA_KEY: testing//testing//testing//testing//testing=
     volumes:
-      - .:/src
+      - .:/src:ro
```

Comment:
Why `ro`? It's nice to have writeable source for development.

Reply:
This is for testing, not development. This is just to make sure it doesn't try to write something and leave broken ownerships all over the place.